config/database.yml

```diff
@@ -79,8 +79,18 @@ test:
 # Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
 # for a full overview on how database connection configuration can be specified.
 #
+# production:
+#   <<: *default
+#   database: board_game_geek_data_production
+#   username: board_game_geek_data
+#   password: <%= ENV["BOARD_GAME_GEEK_DATA_DATABASE_PASSWORD"] %>
+
 production:
```

Comment:
beautiful configuration! glad to see we didn't need the url key added in too. great work!